api/nodes/serializers.py

```diff
@@ -464,6 +467,19 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
     ))
 
     def get_current_user_permissions(self, obj):
+        request = self.context.get('request')
```

Comment:
You use a .get here, but if can't actually get the request you'll have an AttributeError on the next line, should probably check for that or just make this a normal dict lookup.